app/controllers/articles_controller.rb

```diff
@@ -88,14 +88,16 @@ def new
 
   def edit
     authorize @article
+
     @version = @article.has_frontmatter? ? "v1" : "v2"
     @user = @article.user
     @organizations = @user&.organizations
   end
 
   def manage
-    @article = @article.decorate
     authorize @article
+
+    @article = @article.decorate
```

Comment:
Why did you move this line below line 98 rather than leaving it above it on line 97?

Reply:
Because it broke specs :) We passed a decorated object to the authorization layer, though decoration is mostly for controllers and views. I'm not sure what Draper magic was keeping it working before but instead of going down a rabbit hole of fixing it, I just thought "hey, why do we need the decorated object for auth? shouldn't this be just the record". Once I changed that, the spec passed again :D